skopt/space.py

```diff
@@ -42,6 +42,9 @@ class _CategoricalEncoder:
     def __init__(self):
         """Convert labeled categories into one-hot encoded features."""
         self._lb = LabelBinarizer()
+        NONE_ = '__None__'
+        self.encode_none = np.vectorize(lambda a: a if a is not None else NONE_)
+        self.decode_none = np.vectorize(lambda a: a if a != NONE_ else None)
```

Comment:
also there is not really a point in having those as attributes. Just define encode/decode  as regular functions, outside of the class